lua/telescope/builtin/git.lua

```diff
@@ -147,7 +147,7 @@ local set_opts_cwd = function(opts)
   if opts.cwd then
     opts.cwd = vim.fn.expand(opts.cwd)
   else
-    opts.cwd = vim.fn.expand('%:p:h')
+    opts.cwd = vim.fn.getcwd()
```

Comment:
Use `vim.loop.cwd` better, (`vim.fn.getcwd = 0.03267 msecs,  vim.loop.cwd = 009026 msecs`)

Reply:
wasn't aware there's a faster version of cwd in vim.loop, thanks

Reply:
FYI: `vim.fn.expand('%:p:h')` is 0.09 - 0.1 mescs, slowest of them all :smile: